content/millennium/bulk-data/group-export.md

```diff
@@ -0,0 +1,273 @@
+* When the `_type` parameter is not provided, the resources to be exported will be determined by the scopes provided in the authorization token.
+* When the `_typeFilter` parameter is provided, the `_type` parameter MUST be provided as well.
+* `_typeFilter` does not support queries for the following resources: `Location`, `Organization`, `Patient`, `Practitioner`, `Provenance`
+* `_typeFilter` does not support queries with the following search parameters: `_count`, `_contained`, `_containedType`, `_elements`, `_id`, `_include`, `_lastUpdated`, `_revinclude`, `_sort`, `_summary`, `_total`, `encounter`, `identifier`, `patient`, `subject`
+* Supplying `_typeFilter` with the same resource multiple times is not supported.
+* `includeAssociatedData` valid parameters are `LatestProvenanceResources` and `RelevantProvenanceResources`
+* `_outputFormat` valid parameters are `application/fhir+ndjson`, `application/ndjson`, `ndjson`
+* `_outputFormat` defaults to `application/fhir+ndjson` when not provided.
+* The following parameters are not supported: `_elements`, `patient`
+
+_Provenance Behavior_
+
+* Provenances for `Location`, `Organization`, and `Practitioner` will not be exported.
+* When the `_type` parameter is not provided and the token includes the Provenance read scope, all relevant Provenance resources associated with each of the non-provenance resources will be exported by default. You MAY use the  `includeAssociatedData` parameter to return only the latest provenance.
+* When the `_type` parameter is provided with specific resources and includes 'Provenance', all relevant Provenance resources associated with each of the non-provenance resources will be exported by default.
+* When both the `includeAssociatedData` parameter and the `_type` parameter are provided, the `_type` parameter MUST include `Provenance`
+
+### Headers
+
+<%= headers head: {'Accept': 'application/fhir+json', 'Prefer': 'respond-async', Authorization: '&lt;OAuth2 Bearer Token>'} %>
+
+Optional Header
+
+<%= headers head: {'Prefer': 'handling=lenient'} %>
+
+When the Prefer: handling=lenient header is provided any unknown or unsupported parameters will be ignored as specified [here](http://hl7.org/fhir/uv/bulkdata/STU2/export.html#query-parameters).
+
+### Example
+
+Request
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Group/11ec-d16a-c763b73e-98e8-a31715e6a2bf/$export
+
+Response
+
+    Status: 202 Accepted
+    Content-Location: https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/bulk-export/jobs/<job_id>
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Status Request
+
+Checks the status of the export job.
+Use the URL returned in the `Content-Location` header from the kick-off request to retrieve the status of a bulk data export:
+
+    GET /bulk-export/jobs/<job_id>
```

Comment:
I had a comment about making this generic instead of having our current path info as this is not the standard API
something like
`GET <URL from Content-Location header->`